### ISSUE TITLE
2.8x update

### ISSUE THREAD

**Issue by p2or** [p2or/blender-dof-utils#1]
- Module Registration
- Class Registration (Naming, Access)
- [Custom Manipulators](https://devtalk.blender.org/t/linking-custom-manipulators/1140)?

**Comment by DragonHawkAlpha** [comment]
I have created a new fork and updated the code to work with Blender 2.8.  The circles are not being drawn in the correct place so there is still a bit of work to do.

**Comment by p2or** [comment]
Hi @DragonHawkAlpha,

awesome, thanks for your work. Don't have the time right now, but I'll have a look asap. 

Cheers,
Christian

**Comment by DragonHawkAlpha** [comment]
Blender 2.8 RC1 has just been provided on the Blender website.  Now the aperture feature is broken all over again!  Note that there only seems to be f-stop and not radius now.

**Comment by DragonHawkAlpha** [comment]
Now fixed RC1 issues.  It took a whole hour!

**Comment by p2or** [comment]
Hi @DragonHawkAlpha,

thanks for your time fixing all that beta issues. Looks good to me - when you're done could you please provide a pull request along with your name as contributor? Thanks.

Cheers,
Christian

**Comment by DragonHawkAlpha** [comment]
Hi, just added pull request.

**Comment by p2or** [comment]
Hi @DragonHawkAlpha,

merged your PR, many thanks. I fixed some other minor UI issues I found and filed a new release for _Blender 2.8x_, I think that's fine for now. However, there is still a lot to do e.g. filling the geometry with a solid color - I just removed it from the UI for now, personally never used it but I think this can be handy. If you further ideas to improve it, just let me know.

Cheers and thanks again,
Christian
